### PR TITLE
New translation on 1109

### PR DESCRIPTION
**Pull request by HENGRui6**
[oneflow.nn.BCELoss](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/loss.py#L172)
1b8c1857f6015dbfc4bfa0d780c1c72

[oneflow.nn.BCEWithLogitsLoss](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/loss.py#L693)
356d32506a98108d92bddd4329e24cb

[oneflow.nn.BatchNorm1d](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/batchnorm.py#L126)
19a71d5edfc8b9c86b0e208a5f59ab1

[oneflow.nn.BatchNorm2d](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/batchnorm.py#L203)
463b0e13e20c95a361c769647a7dfba

[oneflow.nn.BatchNorm3d](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/batchnorm.py#L278)
484af1575d027754474765dcd3ac208

[oneflow.nn.CELU](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/activation.py#L275)
d229010dcca58c62f54b3ea9053a2cf

[oneflow.Tensor.where](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/where.py#L22)
77157d3bf1557ae4ee1f15b4350faee

[oneflow.Tensor.view](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/reshape.py#L101)
79715dba1c0767a6bb038a4b7f4ad35

[oneflow.Tensor.type_as](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/tensor_ops.py#L24)
163651e937832ddc9ac8f202b88583b

[oneflow.Tensor.topk](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/math_ops.py#L1030)
645960dfb717948b5b97b9656028ac9

[oneflow.Tensor.tolist](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/tensor_ops.py#L229)
46e7ee000fc41fd023ee5278954f935

[oneflow.Tensor.to_local](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/consistent_cast.py#L118)
a98e26eb191025181cce15e6a1d0c41

[oneflow.Tensor.to_consistent](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/consistent_cast.py#L40)
7fe8b3effe74a54540a88987f30bfdc

[oneflow.Tensor.to](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/framework/docstr/tensor.py#L574)
0d0f2e71d154e0d84461dc46feb0aa7

[oneflow.Tensor.tile](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/tile.py#L24)
6edb34ce37c2b76643e216005e37b64

[oneflow.Tensor.sub](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/math_ops.py#L31)
becbadae6ff31f99d4df722eb2d30d6

[oneflow.Tensor.square](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/math_ops.py#L524)
6360915e6cc1e047b19df434b3fcd69

[oneflow.Tensor.sqrt](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/math_ops.py#L498)
e591c554247a3f3d0392afa1c86fae1

[oneflow.Tensor.split](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/split.py#L25)
0539ccffb9f4cf65e5374674db9665d

[oneflow.Tensor.sort](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/sort.py#L47)
c450b71d6a28fb3f53dc3f2bad0d104

[oneflow.Tensor.rsqrt](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/framework/docstr/math_ops.py#L1022)
eb4361f91f78501aa0a24076f1ca5ef

[oneflow.Tensor.long](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/tensor_ops.py#L71)
cc71c02963dfa631f5c8baaefd1dc3c

[oneflow.Tensor.log](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/framework/docstr/math_ops.py#L911)
353d2a3ea45ee344d0ae6504e11a525

[oneflow.Tensor.le](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/comparison.py#L114)
0afdc76612e3e3294180ee73b38a081

[oneflow.Tensor.div](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/framework/docstr/math_ops.py#L81)
1b2803e65be0eac947fa27639215b86

[oneflow.Tensor.cos](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/framework/docstr/math_ops.py#L699)
6a2ee98e6f10f7198ca4963369eff3c

[oneflow.Tensor.cosh](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/framework/docstr/math_ops.py#L725)
97083fb2a961246dddc549272791472

[oneflow.Tensor.add](https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/framework/docstr/math_ops.py#L42)
39a7cbc8b4cab3d65c53fbf6153f249

### CODE REVIEW

**Comment by doombeaker** [comment]
这里已经出现了  conflicts，需要解决下

**Comment by doombeaker** [comment]
这个 PR，由 @YakunZhou 来review，并且一直跟进到此PR合并。

**Comment by YakunZhou** [comment]
https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/loss.py#L716
参数位置没有翻译完整，缺失size_average(bool, optional)  reduse(bool, optional)

**Comment by YakunZhou** [comment]
https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/tensor_ops.py#L23
应为type_as_op

**Comment by YakunZhou** [comment]
https://github.com/Oneflow-Inc/oneflow/blob/52401a6bf801012128fa8e3bc3e576b81fcd6404/python/oneflow/nn/modules/tensor_ops.py#L230
不是GPU 而是CPU

**Comment by S882li** [comment]
**oneflow.Tensor.add, oneflow.Tensor.sub，oneflow.Tensor.div，oneflow.Tensor.le**和**oneflow.Tensor.sub**中与element-wise相关的语句翻译有误。
此外，**oneflow.nn.BatchNorm2d,3d和4d**中，
6J4C8BR5RZNV04XU{XBWRRE
这句话的翻译似乎有些问题，希望能复核一下吧

File: docs/source/cn/loss_ops.py [hunk 1]
```diff
@@ -69,7 +203,7 @@
     .. code-block:: python
 
         >>> import oneflow as flow
-        
+
```
Comment (doombeaker) [inline docs/source/cn/loss_ops.py:206]:
```suggestion

```

不必要的空格

**Comment by doombeaker** [comment]
> **oneflow.Tensor.add, oneflow.Tensor.sub，oneflow.Tensor.div，oneflow.Tensor.le**和**oneflow.Tensor.sub**中与element-wise相关的语句翻译有误。 此外，**oneflow.nn.BatchNorm2d,3d和4d**中， 6J4C8BR5RZNV04XU{XBWRRE 这句话的翻译似乎有些问题，希望能复核一下吧

”与element-wise相关的语句翻译有误。“ 交流越具体精确，越能更快地得到回应/回复。这里指出”有误“，应该还要附上到底是怎么有误，你认为怎样是无误的。

点击 files changed 这个 tab

image

可以具体给PR作者具体某行的翻译 comment，这也昂作者就更能知道是哪里 有误

**Comment by HENGRui6** [comment]
> **oneflow.Tensor.add, oneflow.Tensor.sub，oneflow.Tensor.div，oneflow.Tensor.le**和**oneflow.Tensor.sub**中与element-wise相关的语句翻译有误。 此外，**oneflow.nn.BatchNorm2d,3d和4d**中， 6J4C8BR5RZNV04XU{XBWRRE 这句话的翻译似乎有些问题，希望能复核一下吧

这些element-wise我是按照 https://github.com/Oneflow-Inc/oneflow-api-cn#%E5%BC%80%E5%A7%8B%E7%BF%BB%E8%AF%91 这里翻译的

**Comment by HENGRui6** [comment]
> **oneflow.Tensor.add, oneflow.Tensor.sub，oneflow.Tensor.div，oneflow.Tensor.le**和**oneflow.Tensor.sub**中与element-wise相关的语句翻译有误。 此外，**oneflow.nn.BatchNorm2d,3d和4d**中， 6J4C8BR5RZNV04XU{XBWRRE 这句话的翻译似乎有些问题，希望能复核一下吧

已修改为“此外，默认情况下，在训练期间，该层不断估计计算的均值和方差，然后评估时将其归一化。运行估计默认 :attr:`momentum` 为 0.1 。”

**Review by YakunZhou — APPROVED** [review]
